pipelines/ApPipeWithFakes.yaml

```diff
@@ -59,6 +59,11 @@ tasks:
       connections.coaddName: parameters.coaddName
       connections.fakesType: parameters.fakesType
       doSkySources: True
+      python: |
+        import lsst.meas.extensions.shapeHSM
+        config.measurement.slots.psfShape = "ext_shapeHSM_HsmPsfMoments"
+        config.measurement.slots.shape = "ext_shapeHSM_HsmSourceMoments"
+        config.measurement.plugins.names |= ["ext_shapeHSM_HsmSourceMoments", "ext_shapeHSM_HsmPsfMoments"]
```

Comment:
Same comment here: we should just set this in the Task Config `setDefaults`.

Reply:
Done in the ip_diffim PR. I'm probably going to just cancel/close this PR and delete the branch entirely.